src/index.js

```diff
@@ -0,0 +1,51 @@
+/*  eslint-disable max-classes-per-file, no-useless-catch, no-unused-vars */
+
+import './style.css';
+import createGame from './modules/createGame.js';
+import refreshScores from './modules/refreshScores.js';
+import submitScore from './modules/submitScore.js';
+
+let gameId;
+let isDOMContentLoadedFired = false;
+const msg = document.querySelector('.message');
+const errorMsg = document.querySelector('.error-message');
+
+const createAndInitializeGame = async () => {
+  try {
+    const gameName = 'El Nino';
+    gameId = await createGame(gameName);
+  } catch (error) {
+    errorMsg.textContent = `Failed to create the game : ${error}`;
+    return;
+  }
```

Comment:
Good job connecting your application to the leaderboard API. However, your implementation is not quite correct. Your application creates a new game every time the user submits a score when the page refreshes. This means that scores submitted to the API are not persisted upon page refresh as the application creates a whole new game. Please update your implementation to create a single game outside the application and use the same key returned by the game in all future requests. This way the entire app works with only one game on the API